### ISSUE TITLE
Heracles analyses fails on BigQuery

### ISSUE THREAD

**Issue by acumarav** [OHDSI/WebAPI#1366]
Direct CAST of stratum_1 value to int is not safe and may fail when the query planner decides to execute WHERE clause conditions in parallel. It usually happens on BigQuery.

**Comment by anthonysena** [comment]
@wivern - would it be better to make a 2.7.5 milestone for this item (and for some of the others I see on the rc-2.7.5 branch)?

**Comment by wivern** [comment]
@anthonysena yes, that would be nice to have 2.7.5 milestone
